frontend/components/molecules/Form/FieldSection.js

```diff
@@ -5,15 +5,16 @@ import { SystemComponent } from '../../atoms/SystemComponents';
 import Header4 from '../../atoms/Header4';
 import Input from '../../atoms/Input';
 
-const TitleSection = ({text}) => {
+const TitleSection = ({text, asterik}) => {
     return (
         <SystemComponent>
             <Header4>
                 {text}
+                {asterik && <span style={{color: 'red'}}>*</span>}
```

Comment:
Can you import and use `SystemSpan` instead of <span> ? We are using StyledSystem and Styled-Components extensively throughout this project (instead of regular HTML tags), so using SystemSpan keeps this uniform.